mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/AbstractGenericMojo.java

```diff
@@ -124,6 +130,9 @@ public void initMojo() {
     if (projectBuildDirectory != null) {
       new ProjectDirectoryUpdater(project).updateBuildDirectory(projectBuildDirectory);
     }
+    if (!lightweightPackage) {
```

Comment:
log somehow that if not lightweightpackaging, useLocalRepository will be ignored

Reply:
Sure, will do